src/test/ui/imports/extern-prelude-extern-crate-restricted-shadowing.stderr

```diff
@@ -24,10 +24,10 @@ LL |         extern crate std as Vec;
 LL | define_vec!();
    | -------------- in this macro invocation
 note: `Vec` could also refer to the struct defined here
-  --> $SRC_DIR/std/src/prelude/v1.rs:LL:COL
+  --> $SRC_DIR/std/src/prelude/mod.rs:LL:COL
    |
-LL | pub use crate::vec::Vec;
-   |         ^^^^^^^^^^^^^^^
+LL |     pub use super::v1::*;
+   |             ^^^^^^^^^^^^
```

Comment:
Diagnostic differences aside, I wonder if there is a significant difference between the individual `prelude::rust_20xx` modules containing individual reexports, and them containing glob reexports (I can't think of anything but I also don't know much of the rules). cc @petrochenkov

Reply:
FWIW, there is a difference in how it renders in `rustdoc` (and in that regard, and for the usage here of "extending anothe r module, I personally like the blob more)

Reply:
The only crates where it can theoretically make difference are libcore and libstd (i.e. crates that both define and use a prelude), but in that case we'd immediately see thousands of errors on attempt to build those libraries.